lib/features/mailbox/presentation/mailbox_bindings.dart

```diff
@@ -25,7 +25,7 @@ class MailboxBindings extends Bindings {
     Get.lazyPut<MailboxRepository>(() => Get.find<MailboxRepositoryImpl>());
     Get.lazyPut(() => GetAllMailboxInteractor(Get.find<MailboxRepository>()));
     Get.lazyPut(() => TreeBuilder());
-    Get.lazyPut(() => MailboxController(
+    Get.put(MailboxController(
```

Comment:
why need to change from `lazy` to `put`?

Reply:
We need it to initialize with the same view. Instead of being called when find.